Source/Search/Mixins/AdvancedSearchProviderMixin.lua

```diff
@@ -134,11 +134,14 @@ end
 
 function AuctionatorAdvancedSearchProviderMixin:FilterItemKey(itemKey)
   if itemKey.itemLevel ~= nil and itemKey.itemLevel ~= 0 and self.currentFilter ~= nil then
-    return (self.currentFilter.minItemLevel == nil or self.currentFilter.minItemLevel<=itemKey.itemLevel)
-      and (self.currentFilter.maxItemLevel == nil or self.currentFilter.maxItemLevel>=itemKey.itemLevel)
-  else
-    return true
+    if (self.currentFilter.minItemLevel ~= nil and self.currentFilter.minItemLevel>itemKey.itemLevel) then
```

Comment:
nitpick: please use whitespace between operators for legibility (I'm old and the whitespace helps my eyes 😁 )

Reply:
Still missing whitespace: `self.currentFilter.minItemLevel>itemKey.itemLevel` 😀

Reply:
Parentheses shouldn't be needed either?